### ISSUE TITLE
 node_link_data() got an unexpected keyword argument 'edges'

### ISSUE THREAD

**Issue by adeel-shahzad** [brandondocusen/CntxtPY#2]
I getting below when it is going to save the JSON file,

```
Processing file [162/162]: scrap_job/jobs/web_crawl_job.pys.pyyy.pyyyn.pyys.py.pypy

Completed processing 162 files across 159 directories

Saving graph...

Error: node_link_data() got an unexpected keyword argument 'edges'

Done.

```

**Comment by brandondocusen** [comment]
Solid & safe contribution, thank you!

**Comment by mferris77** [comment]
Sorry to step in to a closed topic, but this actually caused an issue with my setup (first time running it) - I'm on windows.  It appeared to run successfully (parsed 60+ files) and then output an error -  "Error saving graph: node_link_data() got an unexpected keyword argument 'edges'" and then it quit after a timeout.

After I found this thread, I went into the file and removed the 'edges="links"' argument and it ran without issue.

I haven't dug much farther but wanted to pass this on.  Thanks for working on this, looking forward to trying it out!
